src/components/Header/Header.tsx

```diff
@@ -1,19 +1,37 @@
-import { AppBar, Toolbar, Button, Box } from '@mui/material';
-import { useNavigate } from 'react-router-dom';
+import { AppBar, Toolbar, Box, Link } from '@mui/material';
 import Logo from '../Logo/Logo';
 import Navigation from '../Navigation/Navigation';
+import { spacings } from '../DesignSystem/DesignSystem';
+import { Link as RouterLink } from 'react-router-dom';
 
-export default function Header() {
-  const navigate = useNavigate();
+interface HeaderProps {
+  siteTree: {
+    label: string;
+    to: string;
+    onClick?: () => void;
+  }[];
+  headerHeight: number;
+}
 
+export default function Header({ siteTree, headerHeight }: HeaderProps) {
   return (
-    <AppBar component={'header'} position="sticky" color="transparent" elevation={0}>
-      <Toolbar component={'section'}>
-        <Button onClick={() => navigate('/')}>
-          <Logo height={40} />
-        </Button>
+    <AppBar
+      component={'header'}
+      position="sticky"
+      color="transparent"
+      elevation={0}
+      sx={{
+        justifyContent: 'center',
+        background: 'white',
+        height: headerHeight,
+      }}
+    >
+      <Toolbar component={'section'} sx={{ gap: spacings.medium }}>
+        <Link component={RouterLink} to="/" sx={{ height: '70%' }}>
+          <Logo height={'100%'} />
+        </Link>
         <Box flexGrow={1} />
-        <Navigation />
+        <Navigation headerHeight={headerHeight} siteTree={siteTree} />
```

Comment:
Do we really need the headerHeight prop? Looks to me as if this could be solved witin styling keeping the dynamic height..

Reply:
✅ => 🔥 REMOVE `headerHeight` and solve within styling